### PR TITLE
Detect JOBS in a more portable way

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -398,9 +398,8 @@ jobs:
           prepare: |
             pkg_add cmake
           run: |
-            cmake -B build
-            cmake --build build -j $(sysctl -n hw.ncpu)
-            ./build/qjs -qd
+            make
+            make stats
```
Comment (bnoordhuis) [inline .github/workflows/ci.yml]: Just a hunch but make is probably bsd make, not gnu make:
```suggestion
            gmake
            gmake stats
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -413,6 +412,5 @@ jobs:
           prepare: |
             pkg install -y cmake
           run: |
-            cmake -B build
-            cmake --build build -j $(sysctl -n hw.ncpu)
-            ./build/qjs -qd
+            make
+            make stats
```
Comment (bnoordhuis) [inline .github/workflows/ci.yml]:
```suggestion
            gmake
            gmake stats
```

File: .github/workflows/ci.yml (hunk 1 above)
Comment (saghul) [reply .github/workflows/ci.yml]: Yeah it is indeed. TIL `ifeq` is a GNU-ism 😅

**Review by bnoordhuis — APPROVED** [review]

**Comment by saghul** [comment]
WAT there is no way these changes break macOS that way...

**Pull request merged**